src/tools/rados/rados.cc

```diff
@@ -2855,6 +2855,7 @@ static int rados_tool_common(const std::map < std::string, std::string > &opts,
       goto out;
     }
     io_ctx.set_namespace(all_nspaces);
+    io_ctx.set_osdmap_full_try();
```

Comment:
I think we should consider supporting the --force-full option just like the "rm" object command does.